interface/backend/api/serializers.py

```diff
@@ -95,6 +95,9 @@ class Meta:
     def create(self, validated_data):
         return Nodule.objects.get_or_create(candidate=validated_data['candidate'])
 
+    def update(self, instance, validated_data):
+        return super().update(instance, validated_data)
```

Comment:
Why is this necessary? It doesn't overwrite the update method inherited from the parent class, does it?

Reply:
Yes, put it while troubleshooting an other issue and forgot to remove. Thanks for noting!
Will fix the PR tomorrow.